modules/authlinkedin/lib/Auth/Source/LinkedIn.php

```diff
@@ -125,4 +120,23 @@ public function finalStep(&$state) {
 
 		$state['Attributes'] = $attributes;
 	}
+
+    /**
+     * @param $array
```

Comment:
An actual description of what the method does, as well as the parameters and return values, would be helpful here.